### DISCUSSION TITLE
1 Peter, 2 Kings, 2ki, 1sam etc not working

### DISCUSSION THREAD

**Discussion by Tobyrrr00** [tim-hub/obsidian-bible-reference#146] [Q&A]
Hey I'm trying to call 1 Peter 1:1. But it's not working

I've tried a bunch of variations. Even found the internal folder with all the abbreviations and tried a bunch of those too. None of them worked. So every book that starts with 1st 2nd or 3rd doesn't work for me. All others do. Below are some example of what I've tried that doesn't work 
--1Sam1:1
--1 Kgs1:1 
--'1 Kgs'1:1
--1st Kings1:1
--1st Kings 1:1
--'Second Kings1:1'
--1 Kings 1:1

**Comment by tim-hub** [comment]
This should be fixed in release after 2.2.1

Try latest release https://github.com/tim-hub/obsidian-bible-reference/releases/tag/2.3.0

Let me know if you still get any issues?

**Reply by eliplaying** [reply]
Hi, I have the same problem, tested it with 2.3.3 and now 2.3.4, still isn't working :/
(btw: thanks for the awesome plugin!)

**Reply by eliplaying** [reply]
ok i checked, it is working with the "English Word Bible @Bible API", but not with the "German Deutsch Elberfelder Bibel @Bolls Life" or the "German Luther", I checked with Polish translation @Bolls Life, it also works, so maybe it's just an issue with the two German translations atm.

**Reply by tim-hub** [reply]
Can you share the trigger `book name, chapter/verse number?`
